src/components/about/About.component.tsx

```diff
@@ -0,0 +1,18 @@
+import about from '@/assets/about/index.png';
```

Comment:
Please add some decent names. Like
```suggestion
import aboutImg from '@/assets/about/about-photo.png';
```

Reply:
Fixed